playground/README.md

```
@@ -0,0 +1,56 @@
+   cp src/default.playground.ts src/my-test.playground.ts
+   ```
+
+2. Edit your new file with your test code.
+
+3. Refresh the browser - your new playground appears in the dropdown.
+
+4. Select it from the dropdown to run it.
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)